Makefile

```diff
@@ -8,7 +8,7 @@ NAMESPACE	?= elastic
 TIMEOUT := 1200s
 
 CHART_NAME ?= elastic/elasticsearch
-CHART_VERSION ?= 7.9.3
+CHART_VERSION ?= 7.15.0
```

Comment:
Wouldn't that also upgrade the ElasticSearch version? Our plugin only supports till `7.9.3`.

Reply:
Nah it won’t. The application image is pinned here: https://github.com/greenpeace/planet4-helm-elasticsearch/blob/3ff02341c25dc2d61ff9124f87bd40998ab28747/values.yaml#L4

This overrides the default updated image in the helm chart so we end up with updated manifests (newer apiversions) but the application version stays the same.